hardhat/contracts/xchain/AccountCache.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Should the whole `xchain` be moved under `@oasisprotocol/sapphire-paratime/opl`?